boxoffice/models/order.py

```diff
@@ -89,3 +89,14 @@ def get_amounts(self):
             discounted_amount += line_item.discounted_amount
             final_amount += line_item.final_amount
         return order_amounts_ntuple(base_amount, discounted_amount, final_amount)
+
+    @property
+    def is_confirmed(self):
+        return self.status in [ORDER_STATUS.SALES_ORDER, ORDER_STATUS.INVOICE]
```

Comment:
Add this list to `ORDER_STATUS` so you can change this to `return self.status in ORDER_STATUS.CONFIRMED`.